### PR TITLE
plugin: fzopen: print full path for picker mode

### PR DESCRIPTION
**Pull request by N-R-K**
Closes: https://github.com/mcchrish/nnn.vim/issues/133

@jarun Since I don't use the plugin IDK if this might have any side-effects.

### CODE REVIEW

**Comment by N-R-K** [comment]
Am also assuming that `$entry` is only filename and not fullpath. If entry is fullpath somehow then this wouldn't work.

File: plugins/fzopen [hunk 1]
```diff
@@ -42,9 +42,9 @@ fi
 if [ "$3" ]; then
     if [ "$entry" ]; then
         if [ "-" = "$3" ]; then
-            printf "%s\n" "$entry"
+            printf "%s\n" "${2}/$entry"
```
Comment (jarun) [inline plugins/fzopen:45]: I would say don't assume anything and check if the path starts with `/` or not. If not, assume it's relative to the current directory (where fzopen is issued).

**Review by jarun — CHANGES_REQUESTED** [review]

File: plugins/fzopen (hunk 1 above)
Comment (N-R-K) [reply plugins/fzopen:45]: > If not, assume it's relative to the current directory (where fzopen is issued).

Should I use `$2` or `$PWD` then?

Comment (jarun) [reply plugins/fzopen]: I think `$PWD` would be the right thing to use.

**Comment by N-R-K** [comment]
Okay, we should be good. Just need a regular user of this plugin to test and confirm this doesn't break anything.

**Comment by jarun** [comment]
Thank you!

**Pull request merged**